### PR TITLE
Fix wpe view backend leak

### PR DESCRIPTION
**Pull request by emutavchi**

### CODE REVIEW

File: Source/WebKit/UIProcess/API/wpe/WPEView.cpp [hunk 1]
```diff
@@ -152,6 +152,7 @@ View::View(struct wpe_view_backend* backend, const API::PageConfiguration& baseC
 View::~View()
 {
     m_compositingManagerProxy.finalize();
+    wpe_view_backend_destroy(m_backend);
```
Comment (zdobersek) [inline Source/WebKit/UIProcess/API/wpe/WPEView.cpp:155]: The backend ownership is not intended for this class. Sure this fixes the leak, but the ownership should be managed alongside the WKView ownership.

**Review by zdobersek — CHANGES_REQUESTED** [review]

File: Source/WebKit/UIProcess/API/wpe/WPEView.cpp (hunk 1 above)
Comment (zdobersek) [reply Source/WebKit/UIProcess/API/wpe/WPEView.cpp:155]: i.e. this change would not be necessary, instead the wpe_view_backend lifetime should be managed inside the plugin.

Comment (emutavchi) [reply Source/WebKit/UIProcess/API/wpe/WPEView.cpp:155]: @zdobersek  this change makes wpe-2.22 behave similarly to wpe-20170728. So, my understanding is that from wpe-2.22 the WKViewCreate user takes ownership over the backend, is that correct?

Comment (zdobersek) [reply Source/WebKit/UIProcess/API/wpe/WPEView.cpp:155]: OK, I think this should be fine for the WK2 C API. Let me just check before merging.

**Review by zdobersek — APPROVED** [review]

**Pull request merged**